src/views/mainLayout.js

```diff
@@ -52,16 +73,20 @@ const mainLayout = data => {
       </head>
       <body>
         <header>
-          <a href='/user/breaches'>
-            <img class='monitor-logo' srcset='/images/monitor-logo-transparent.webp 213w, /images/monitor-logo-transparent@2x.webp 425w' width='213' height='33' alt='${getMessage('brand-fx-monitor')}'>
-          </a>
-          <div class='nav-wrapper'>
-            <button class='nav-toggle'>
-              <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 8' width='20'>
-                <path d='M1 1h8M1 4h8M1 7h8' stroke='#000' stroke-width='1' stroke-linecap='round'/>
-              </svg>
-            </button>
-            ${userMenu(data)}
+          <!-- Cancel Premium Subscription -->
+          ${shouldShowRecruitmentBanner() ? recruitmentBanner() : ''}
```

Comment:
Conversely, the `recruitmentBanner()` fn could stealthily call `shouldShowRecuitmentBanner()` and return markup or an empty string.

Reply:
Done! Great suggestion. Revised in ad11567d72b884d0a90d9cd5b9a365d29d1569cb